packages/typespec-ts/test/modularUnit/scenarios/apiOperations/apiOperations.md

```diff
@@ -1161,3 +1161,316 @@ export class TestingClient {
+  @visibility(Lifecycle.Create, Lifecycle.Read, Lifecycle.Update)
+  @doc("The provisioned IO / sec of the share.")
+  provisionedIOPerSec?: int32;
+
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "customized style suitable for storage admins"
+  @visibility(Lifecycle.Read)
+  @doc("A date/time value that specifies when the provisioned IOPS for the file share is permitted to be reduced.")
+  provisionedIOPerSecNextAllowedDowngrade?: utcDateTime;
+
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "customized style suitable for storage admins"
+  @visibility(Lifecycle.Create, Lifecycle.Read, Lifecycle.Update)
+  @doc("The provisioned throughput / sec of the share.")
+  provisionedThroughputMiBPerSec?: int32;
+
+  @visibility(Lifecycle.Read)
+  @doc("A date/time value that specifies when the provisioned throughput for the file share is permitted to be reduced.")
+  provisionedThroughputNextAllowedDowngrade?: utcDateTime;
+
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "customized style suitable for storage admins"
+  @visibility(Lifecycle.Read)
+  @minValue(3000)
+  @doc("Burst IOPS are extra buffer IOPS enabling you to consume more than your provisioned IOPS for a short period of time, depending on the burst credits available for your share.")
+  includedBurstIOPerSec?: int32;
+
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "customized style suitable for storage admins"
+  @visibility(Lifecycle.Read)
+  @minValue(0)
+  @doc("Max burst IOPS credits shows the maximum number of burst credits the share can have at the current IOPS provisioning level.")
+  maxBurstIOPerSecCredits?: int64;
+
+}
+
+@doc("File share resource")
+model FileShare is Azure.ResourceManager.TrackedResource<FileShareProperties> {
+  @doc("The resource name of the file share, as seen by the administrator through Azure Resource Manager.")
+  @pattern("^([a-z]|[0-9])([a-z]|[0-9]|(-(?!-))){1,61}([a-z]|[0-9])$")
+  @key("resourceName")
+  @path
+  @segment("fileShares")
+  name: string;
+}
+
+
+model FileShareSnapshotProperties {
+  // The FileShareSnapshot time of the file share FileShareSnapshot.
+  @visibility(Lifecycle.Read)
+  @doc("The FileShareSnapshot time in UTC in string representation")
+  snapshotTime?: string;
+
+  // User-defined value. Resource users and/or automated systems (like Azure Backup) managing snapshots will specify the value based on semantics of their own.
+  // Examples:
+  // - Users in Company A might use aliases.
+  // - Users in company B might use email addresses.
+  // - Automated systems like MAB might use resource identifier of Azure Backup Vault.
+  @visibility(Lifecycle.Read)
+  @doc("The initiator of the FileShareSnapshot. This is a user-defined value.")
+  initiatorId?: string;
+
+  @doc("The metadata")
+  metadata?: Record<string>;
+}
+
+@armResourceOperations
+interface FileShareSnapshots {
+ updateFileShareSnapshot is ArmCustomPatchAsync<
```

Comment:
could you simply this case? there are too many properties and comments. i don't think we need these noises.